test/dynamics/models/test_rotating_frame.py

```diff
@@ -486,7 +489,8 @@ def test_vectorized_frame_basis(self):
         self.assertAllClose(op1, op2.reshape((6, 6), order="F"))
 
 
-class TestRotatingFrameTypeHandling(QiskitDynamicsTestCase):
+@partial(test_array_backends, array_libraries=["numpy", "jax"])
```

Comment:
Does this class actually need both `"numpy"` and `"jax"`? I think it's okay for this to only use numpy.

Reply:
fixed at https://github.com/Qiskit-Extensions/qiskit-dynamics/pull/290/commits/6ca86b6f1bc9c3f0fca7d5b58f4aff14b01495e3.